### DISCUSSION TITLE
Trying to capitalise first letter of sentences

### DISCUSSION THREAD

**Discussion by barha123** [No3371/obsidian-regex-pipeline#32] [Q&A]
Hello! I am trying to capitalise the first letters of sentences using the following expression:

"(?<=^|[.!?]\s|\n)([a-zA-Z])(?!\.)"
->
"\U$1"

Doesnt seem to work, it return \U before the expression it caught (the equivalent to $1)
For instance:

aaaa. bbbb, cccc
will return \Uaaaa. \Ubbbb, cccc

Thanks for helping

**Comment by No3371** [comment]
I'm not very sure about how Electron app works but It seems like the \U and \L support is not included in Obsidian runtime. 
It does not work in RegExr too, but works in VSCode.
I guess it's not very universally supported.
